src/lib/airtable/patches/README.md

```diff
@@ -1,10 +1,21 @@
 ### Overview
 This folder contains patches that should be run after regenerating the airtable schema. This is needed to maintain consistency between the frontend and backend.
 
-Add additional patches here via `git format-patch ... -o <path/to/dir>`.
-For more information, you can view the resource [here](https://devconnected.com/how-to-create-and-apply-git-patch-files/)
+### Overview
+This folder contains patches that should be run after regenerating the airtable schema. This is needed to maintain consistency between the frontend and backend.
+
+## Add Additional Patches
```

Comment:
UX nit: maybe swap this with **Directions to apply** since I'm assuming that part is most relevant?